radio/src/datastructs.h

```diff
@@ -73,7 +73,7 @@ static inline void check_struct()
   CHKSIZE(TimerData, 17);
   CHKSIZE(ModelHeader, 131);
   CHKSIZE(CustomScreenData, 1892);
-  #if defined(PCBNV14) || defined(RADIO_NB4P)
+  #if defined(PCBNV14) || defined(RADIO_NB4P) || defined(RADIO_NV14_FAMILY)
```

Comment:
```suggestion
  #if defined(RADIO_NB4P) || defined(RADIO_NV14_FAMILY)
```
Think you can drop the PCBNV14 here

Reply:
Nope, not now otherwise you cannot use the old target.  Need to do another cleanup in next big release.

Reply:
Ah, ok, I missed the point that this was keeping the old targets for now, makes sense now. Sorry for the noise there. Usability tests will be tomorrow, and I'll merge if nothing arises.